modules/ROOT/pages/ldap/ldap-connector.adoc

```diff
@@ -272,7 +276,7 @@ config.url=<URL>
 |===
 |Parameter|Value
 |Display Name|HTTP
-|Extension configuration| If no HTTP element has been created yet, click the plus sign to add a new HTTP Listener Configuration and click OK (leave the values to its defaults).
+|Connector configuration| If no HTTP element has been created yet, click the plus sign to add a new HTTP Listener Configuration and click OK (leave the values to its defaults).
```

Comment:
click *OK*...

I would take the text in the parentheses and make a sentence out of it.